src/features/multivariate_layer/components/MultivariateAnalysisForm/MultivariateAnalysisForm.tsx

```
@@ -135,6 +145,13 @@ export function MultivariateAnalysisForm({
             })),
           }
         : null;
+    // mcda opacity takes precedence
+    let opacity: number | MCDALayer[] | undefined = dimensionsLayers.opacity;
+    if (!opacity.length && !!opacityStatic) {
+      opacity = isNumber(parseFloat(opacityStatic))
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)